content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -422,6 +439,14 @@ However, configuring the proxy this way does require cluster provider specific k
 In addition to that, you loose monitoring of access to external services and cannot apply Istio features on traffic to
 external services.
 
+The third approach directs the traffic through the Istio sidecar proxy, however it allows access to any service on any
+ports that have no HTTP service inside the mesh and no service entry defined. In this approach, similarly to the
+previous one, you cannot monitor access to external services. The advantage of this approach is that you must not
```

Comment:
is that you don't need to know which ...

Reply:
Fixed in https://github.com/istio/istio.io/pull/3222/commits/f2b596a392a6193f44132f9d7c8e9915ca63b33a